client/changelist/file_changelist_test.go

```diff
@@ -78,3 +78,47 @@ func TestListOrder(t *testing.T) {
 	assert.Equal(t, c2.Path(), cs[1].Path(), "Path 2 mismatch")
 	assert.Equal(t, c2.Content(), cs[1].Content(), "Content 2 mismatch")
 }
+
+func TestFileChangeIterator(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("/tmp", "test")
+	defer os.RemoveAll(tmpDir)
```

Comment:
Looks like we have this same defer at line 89. One of them should be removed. Generally I put the defer after the error because the cleanup shouldn't be required if the command didn't complete successfully.